### PR TITLE
Syndicate suit and tie bundle

### PR DESCRIPTION
**Pull request by JoeHammad1844**
## About the PR

adds a suit and tie bundle into the misc tab because I listened to justin timberlake's suit and tie

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: JoeHammad
- add: Syndicate agents may now purchase a snazzy suit and tie!

### CODE REVIEW

File: Resources/Prototypes/Catalog/Fills/Boxes/general.yml [hunk 1]
```diff
@@ -414,3 +414,17 @@
...
+  id: BoxSuitAndTie
+  description: A cardboard box for storing things.
+  components:
+  - type: StorageFill
+    contents:
+      - id: ClothingEyesGlassesSunglasses
+      - id: ClothingNeckTieRed
+      - id: ClothingHandsGlovesLatex
+      - id: ClothingUniformJumpsuitLawyerBlack
+      - id: ClothingShoesBootsLaceup
```
Comment (crazybrain23) [inline Resources/Prototypes/Catalog/Fills/Boxes/general.yml:430]: Needs newline at end of file.

File: Resources/Prototypes/Catalog/uplink_catalog.yml [hunk 2]
```diff
@@ -1194,6 +1194,16 @@
   categories:
     - UplinkMisc
 
+- type: listing
```
Comment (crazybrain23) [inline Resources/Prototypes/Catalog/uplink_catalog.yml:1197]: Should be in pointless category.

File: Resources/Locale/en-US/store/uplink-catalog.ftl [hunk 3]
```diff
@@ -282,6 +282,9 @@ uplink-syndicate-segway-crate-desc = Be an enemy of the corporation, in style!
 uplink-syndicate-sponge-box-name = Syndicate Sponge Box
 uplink-syndicate-sponge-box-desc = A box containing 6 syndicate sponges disguised as monkey cubes, these cubes turn into a variety of angry wildlife after coming into contact with water.
 
+uplink-suitandtie-name = Suit and tie bundle
```
Comment (crazybrain23) [inline Resources/Locale/en-US/store/uplink-catalog.ftl:285]: Should be in pointless category.

**Review by crazybrain23 — CHANGES_REQUESTED** [review]
I'm not a maintainer, so you can ignore this unless they agree with me.

**Comment by JoeHammad1844** [comment]
it comes with a pair of sunglasses, which block flashes, so pointless doesn't feel right

File: Resources/Prototypes/Catalog/Fills/Boxes/general.yml [hunk 4]
```diff
@@ -414,3 +414,18 @@
     layers:
       - state: box
       - state: syringe
+
+- type: entity
+  name: cardboard box
+  parent: BoxCardboard
+  id: BoxSuitAndTie
+  description: A cardboard box for storing things.
```
Comment (deltanedas) [inline Resources/Prototypes/Catalog/Fills/Boxes/general.yml:422]: why duplicate name and description

File: Resources/Prototypes/Catalog/uplink_catalog.yml [hunk 5]
```diff
@@ -1194,6 +1194,16 @@
...
+  id: UplinkSuitAndTie
+  name: uplink-suitandtie-name
+  description: uplink-suitandtie-desc
+  productEntity: BoxSuitAndTie
+  cost:
+    Telecrystal: 1
```
Comment (deltanedas) [inline Resources/Prototypes/Catalog/uplink_catalog.yml:1203]: flash protection too cheap for 1 tc (mask is 2 tc iirc) remove the sunglasses or up the price

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Catalog/uplink_catalog.yml (hunk 5 above)
Comment (JoeHammad1844) [reply Resources/Prototypes/Catalog/uplink_catalog.yml:1203]: SIR YES SIR OORAH

**Comment by deltanedas** [comment]
now that it really is pointless put it in pointless category with the rest of the drip

**Review by crazybrain23 — APPROVED** [review]

**Comment by JoeHammad1844** [comment]
ill just draft this until the fate of #18905 is decided

**Pull request closed without merging**